vendor/cran.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
Agreed that `services` is preferable to `vendor`… more descriptive and avoids the name clash.

I don't tend to use a `test/` tree or `tests` directories, preferring to store the tests right next to the code. It relates to [grouping by coupling, not by function, and storing tests next to the code](https://github.com/focusaurus/express_code_structure#underlying-principles-and-motivations) and reduces friction. There is nothing but tests for services, so another option would be a `service-tests` folder at the top level, until that changes.

Would prefer to break up what is in `test/` than to add more.